src/main/scala/PcapExample.scala

```diff
@@ -69,14 +63,16 @@ object PcapExample {
         !(data + offsetBytes + n)
       }
       .foreach { v =>
-        native.stdio.printf(toCString("%02X"), v)
+        native.stdio.printf(c"%02X", v)
       }
     println("...]")
   }
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = Zone { implicit zone => run(args) }
+
+  def run(args: Array[String])(implicit zone: Zone): Unit = {
     val cooked = args.contains("cooked")
-    val live = args.contains("live")
+    val live = !cooked
```

Comment:
The following logic says that if "live" is not given, then it assumes "cooked", and reads the last (non existent) argument.  That suggested to me that it was a binary choice.